11-test-react-mastery/src/css/App.css

```diff
@@ -0,0 +1,7 @@
+.custom {
```

Comment:
Created custom class in css for background

Reply:
just changed this to a more descriptive class name and we're good

Reply:
Done, used a descriptive class name.